metricbeat/module/logstash/logstash_test.go

```diff
@@ -15,12 +15,19 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package logstash
+package logstash_test
```

Comment:
Out of curiosity, was it a problem to use the original package names?

Reply:
Without this there's an import cycle because of the metricsets' imports below.

Reply:
Oh ok, because metricsets import this package :+1: